evalml/pipelines/component_graph.py

```diff
@@ -30,6 +30,20 @@ def __init__(self, component_dict=None, random_seed=0):
         self._feature_provenance = {}
         self._i = 0
 
+    @property
+    def default_parameters(self):
+        """The default parameter dictionary for this pipeline.
+
+        Returns:
+            dict: Dictionary of all component default parameters.
+        """
+        defaults = {}
+        for c in self.component_instances:
```

Comment:
nit-pick: I think this would work a be a bit clearer

```python
        defaults = {}
        for component in self.component_instances.values():
            if component.default_parameters:
                defaults[component.name] = component.default_parameters
        return defaults
```

Reply:
100% agreed, tyty!